torchaudio/functional/functional.py

```diff
@@ -994,6 +1000,53 @@ def spectral_centroid(
     return (freqs * specgram).sum(dim=freq_dim) / specgram.sum(dim=freq_dim)
 
 
+@_mod_utils.requires_module('torchaudio._torchaudio')
+def apply_codec(
+    waveform: Tensor,
+    sample_rate: int,
+    format: str,
+    compression: Optional[float] = None,
+    channels_first: bool = True,
+    encoding: Optional[str] = None,
+    bits_per_sample: Optional[int] = None,
```

Comment:
Can you reorder the parameters as `waveform, sample_rate, channels_first, format, encoding, bits_per_sample`?

Sorry I am reverting what I originally asked (moving `channels_first` after `format`) but grouping format related parameters together looks nicer.

Reply:
Moving the `channels_first` (default_parameter) before `format` (non default_parameter) throws a syntax-error ```SyntaxError: non-default argument follows default argument```.
Also what about the `compression` parameter?

Reply:
Sorry, I made an invalid suggestion again.
let's do `waveform, sample_rate, format, channels_first, compression, encoding, bits_per_sample`